.seed-config.yaml

```diff
@@ -27,7 +27,11 @@ supportLinks:
 
 readme:
   usageExample: |
-    First you need to install Tor on your machine. Take a look at the [Tor download instructions](https://www.torproject.org/download/)
+    Note: The `master` branch is using [Caddy v2](https://caddyserver.com/), if you
+    want to use Torproxy with previous caddy versions, check the `caddy-v1` branch.
+
+    If you want Torproxy to start a fresh instance of Tor, you need to install Tor
+    on your machine. Take a look at the [Tor download instructions](https://www.torproject.org/download/)
 
     You can get torproxy as a plugin on caddy's [build server](https://caddyserver.com/download).
```

Comment:
The build server comment here should note this would build the v2 version.
Also the link is different, but redirects to v1. So should be fine.